governance/polls/SparkLend Ethereum - Reactivate WBTC and Optimize Parameters for Current Market Conditions - November 6, 2023.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SparkLend Ethereum - Reactivate WBTC and Optimize Parameters for Current Market Conditions - November 6, 2023
+summary: Signal your support or opposition to reactivating WBTC on SparkLend Ethereum and optimizing parameters for current market conditions.
+discussion_link: https://forum.makerdao.com/t/proposal-to-adjust-sparklend-parameters/22542
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-11-06T16:00:00
+end_date: 2023-11-09T16:00:00
+
+
+---
+
+# Poll: SparkLend Ethereum - Reactivate WBTC and Optimize Parameters for Current Market Conditions - November 6, 2023
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of Spark Protocol and the Stability Scope Facilitator. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, November 6 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+WBTC has proven itself to be an asset that can scale, and decentralized collateral requirements have been relaxed slightly. Phoenix Labs is therefore proposing to reactivate (unfreeze) the WBTC market and optimize parameters for current market conditions. 
```

Comment:
I'll make an adjustment, thanks VW! Generally, the statement is backed by data that shows the performance of the asset. And we are providing a small amount of context to summarize that data so that the community doesn't need to ingest the BA Labs data. But I will tone down the language a bit.

Reply:
@VoteWizard edit done and ready for your review!